README.md

```diff
@@ -1,60 +1,37 @@
-# SuperX
-## Team members:
-### 1. Oded Hellman
-### 2. Yoav Ben Hur
-### 3. Aryeh Klein
-### 4. Tzachi Elrom
-### 5. Shai Brown
+![superx](superx/static/img/superx.JPG)
 
-________________________________
+## **Main Goal** 
+**Comparing prices between supermarkets by using accesible data- to save money**
+- To activate and use the app- please follow the instructions in [User's_Guide file](User's_Guide.md)
 
-## Initial System Spesification:
+## Project Architecture
+The project runs on Vagrant VM & uses flask as a web framework.
+To create the project's envrionment- bootstrap.sh file is triggered when the app starts running.
 
-**SuperX purpose:** price comparison between supermarkets in Israel
-  
-#### User Story:
-1. User logs in to the website and chooses the relevant grocery store
-2. The user enters "Prices Comparison" button
-3. A get request is sent to the prices' table in the server
-4. The user gets a list of supermarkets' prices around him, cheapest first
-*There are details about the prices of each product
-5. The user gets an option to order the grocery list directly from the wanted supermarket by clicking on the "order" button
+### **Frontend** 
+Client UI to choose the products to compare and the city where the client lives. <br>
+<u>Languages</u>- HTML, CSS, JS
 
-*There's an option to show the supermarkets that are in a certain radius from user's location
+### **Backend**
+Information extractors scripts that contact the relevant urls and extract the relevant data to the database.<br>
```

Comment:
Information extractor scripts that web scrape supermarket urls, extract the relevant information and places it into the db